test/projectile-test.el

```diff
@@ -720,6 +720,26 @@
         (should (equal (list (expand-file-name "vendor/client-submodule/" project))
                        (projectile-get-all-sub-projects project)))))))
 
+(ert-deftest projectile-test-projectile-shell-command-to-string-fallback ()
+  (let ((command "command arg1 arg2")
+        (command-path "/path/to/command")
+        shell-command-args call-process-args)
+    (noflet ((shell-command-to-string (&rest args)
```

Comment:
The noflet indentation seems broken. Probably you didn't load `noflet` before indenting this buffer.